.github/workflows/js-lint-test.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   lint:
     name:    JS linting
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
The `ubuntu:latest` tag points to the "latest LTS", since that's the version recommended for general use